### PR TITLE
Building your own decays lesson

### PR DESCRIPTION
**Pull request by apuignav**
This may end up split in several sub-lessons.

### CODE REVIEW

**Comment by apuignav** [comment]
The lessons have changed significantly, so a few extra pairs of eyes would be nice.

**Review by lcapriot — CHANGES_REQUESTED** [review]
Thanks Albert! My comments:

In building-decays-part0.md
- The definition of Selection is a bit confusing. The first thing you say is that "It uses other Selections to process LHCb::Particles...". Does it make sense to say simply "It processes LHCb::Particles..." ?

In building-decays-part1.md
- dstar_sel should have d0_sel as RequiredSelection and not dstar

File: building-decays-part1.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    DecayDescriptor='[D0 -> pi- K+]cc',
+    DaughtersCuts=dstar_daughters,
+    CombinationCut=dstar_comb,
+    MotherCut=dstar_mother
+)
+
+dstar_sel = Selection(
+    'Sel_Dstar',
+    Algorithm=dstar,
+    RequiredSelections=[dstar, Pions]
```
Comment (lcapriot) [inline building-decays-part1.md]: RequiredSelections=[d0_sel, Pions]

Comment (apuignav) [reply building-decays-part1.md]: Thanks! Added.

**Pull request merged**